dpgv4.py

```diff
@@ -364,7 +366,7 @@ def video_quality_options() -> Mapping[int, Iterable[str]]:
         "-trellis", "1",
         "-mpv_flags", "+cbp_rd",
         "-mpv_flags", "+mv0",
-        "-b:v", "256k",
+        "-b:v", "512k",
```

Comment:
This change is not acceptable for me because of the output file size increase it entails. Please make it dependent on high quality mode.